backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/util/FreinetApi.kt

```diff
@@ -135,13 +132,63 @@ class FreinetApi(private val host: String, private val accessKey: String, privat
                     contentType(ContentType.Application.Json)
                     setBody(requestBody)
                 }
-                val body = response.bodyAsText()
-                logger.devInfo("Successfully created person in freinet: $body")
+                logger.error("Successfully created person in freinet: ${response.bodyAsText()}")
```

Comment:
```suggestion
                logger. devInfo("Successfully created person in freinet: ${response.bodyAsText()}")
```
why changed to error?

Reply:
yes. thx :)